Changelog.md

```diff
@@ -2,6 +2,19 @@
 
 * motoko (`moc`)
 
+  * BREAKING CHANGE
+
+    Motoko Candid de-serialization now checks that the actual type of a deserialized actor or function reference is a subtype of the expected type (#3171),
```

Comment:
It was checking subtype before, now we *only* check reference types. But this whole thing probably doesn't make sense for external developers. Maybe just say we implement Candid spec 1.4?

Reply:
Actually, no. Motoko was never checking subtyping at all, even at option types. We skipped that version of Candid.

Reply:
But yeah, maybe this is too much information.

Reply:
Ah okay, then this changelog makes sense, but probably too detailed, even I didn't know about this :)